lang_python.adoc

```diff
@@ -13,130 +13,171 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 `-t python` option:
 
 [source,shell]
-kaitai-struct-compiler -t python my_spec.ksy
+kaitai-struct-compiler -t python --python-package . my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use Python 3
+because Python 2 is EOL since 2020-01-01. You can also install
+`python-is-python3` on Ubuntu focal to make `python3` be used via
+`python` command.
+
+`--python-package` optional CLI arg allows you to select a parent package
+name used by one generated file to import symbols from another one.
+
+If you **don't intend to embed** the KSC-generated files and just want to
+play a bit in IPython/Jupyter shell in the same dir, **remove** this arg,
```

Comment:
I would use either "REPL" or "Python shell" here, because this also applies when using the default interactive Python shell, not just IPython.

Reply:
Thanks, fixed.